db/background_mgr_resync_dcp_test.go

```diff
@@ -336,6 +336,43 @@ func TestResyncManagerDCPRunTwice(t *testing.T) {
 	wg.Wait()
 }
 
+func TestResyncImportComputeStat(t *testing.T) {
+	if base.UnitTestUrlIsWalrus() {
+		t.Skip("Test requires Couchbase Server")
+	}
+	const docsToCreate = 1000
```

Comment:
can we create fewer docs? creating a lot of docs causes them to show up in future tests DCP feeds as deleted until the server decides to reset DCP feed. We should need minimal docs for this test.

It probably won't make this test faster but it will make some other tests fast.